### ISSUE TITLE
Filebot broken - java installer 2.13

### ISSUE THREAD

**Issue by DKeppi** [rednoah/java-installer#18]
After installing the new java installer 2.13 on my Synology, filebot doesnt work anymore!

```
BUG! exception in phase 'semantic analysis' in source unit 'Script1.groovy' Unsupported class file major version 57
BUG! exception in phase 'semantic analysis' in source unit 'Script1.groovy' Unsupported class file major version 57
	at net.filebot.cli.ScriptShell.evaluate(ScriptShell.java:64)
	at net.filebot.cli.ScriptShell.runScript(ScriptShell.java:74)
	at net.filebot.cli.ArgumentProcessor.runScript(ArgumentProcessor.java:163)
	at net.filebot.cli.ArgumentProcessor.run(ArgumentProcessor.java:37)
	at net.filebot.Main.main(Main.java:132)
Caused by: java.lang.IllegalArgumentException: Unsupported class file major version 57
	at groovyjarjarasm.asm.ClassReader.<init>(ClassReader.java:184)
	at groovyjarjarasm.asm.ClassReader.<init>(ClassReader.java:166)
	at groovyjarjarasm.asm.ClassReader.<init>(ClassReader.java:152)
	at groovyjarjarasm.asm.ClassReader.<init>(ClassReader.java:273)
	... 5 more

Error (o_O)
```

**Comment by DKeppi** [comment]
Reinstall of the old package doesnt help...same problem!
Is it a java issue with actual version?

**Comment by rednoah** [comment]
Looks like `FileBot 4.8.5` is not forwards-compatible with `Java 13`:
https://www.filebot.net/forums/viewtopic.php?t=11099

TL;DR it's a FileBot issue

**Comment by DKeppi** [comment]
THX
